superset/requirements-dev.txt

```diff
@@ -25,7 +25,7 @@ mysqlclient==1.3.13
 pip-tools==3.6.1
 psycopg2-binary==2.7.5
 pycodestyle==2.4.0
-pyhive==0.6.1
+-e git+https://github.com/src-d/PyHive.git@master#egg=PyHive
```

Comment:
should we make a tag in our fork, instead of using master?

Reply:
DP team is responsible for that repo. So I don't know. I work with what I have.

Reply:
cc @ajnavarro maybe we should tag something like `v0.6.1-srcd1`?

Reply:
https://github.com/src-d/PyHive/releases/tag/v0.7.0-srcd1